templates/query/.template.config/template.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Just an observation, but a lot of template authors keep this template file separate in a private repo. I don't know why and not suggesting we should do that. Just raising it in case someone else understands why this seems to be the convention and if it's something we should do.

Reply:
Thanks for the feedback, I wasn't aware of that.  I think it makes sense to keep in the main repo for now, so that it's included  for anyone who installs the template.  Will keep this in mind though for future work.